### ISSUE TITLE
Why there is no bias term?

### ISSUE THREAD

**Issue by cmusjtuliuyuan** [intel/BigDL-Tutorials#6]
There is no bias term in linear_regression.ipynb, deep_feed_forward_neural_network.ipynb, cnn.ipynob ... 
I don't understand why?

For example, in the BigDL tutorials:
` model.add(Linear(n_input, n_hidden_1).set_name('mlp_fc1'))
`
` model.add(ReLU())
`
When these two lines implemened in Tensorflow or Pytorch, it will look like:
` W = weight_variable([input_dim, output_dim])
`
`b = bias_variable([output_dim])
`
`logits=tf.nn.relu(tf.matmul(input_placeholder, W) + b)
`

**Comment by cmusjtuliuyuan** [comment]
I understand the bias term is included into the Linear
